docs/CONNECTION.md

```diff
@@ -0,0 +1,29 @@
+# Connecting to an Enervent unit
+
+These instructions are based on:
+
+* https://doc.enervent.com/op/op.ViewOnline.php?documentid=999&version=1 (Freeway WEB-väyläsovitin - Asennus- ja käyttöohjeet)
+* https://web.archive.org/web/20201020102005/http://ala-paavola.fi/jaakko/doku.php?id=pingvin
+
+Please read the above documents before continuing!
+
+## Requirements
+
+* A 4P4C cable (sometimes called RJ10)
+* An RS-485 device
```

Comment:
*knitpicking*
A RS-485 device

Reply:
AFAIK no, this is correct. Since it's pronounced "arr ess 485" it should be with an "an".

Reply:
Ah, refreshing my grammars here :D Thank you for the lesson :D